internal/nonkube/bootstrap/install.go

```diff
@@ -16,5 +39,261 @@ func Install(platform string) error {
 		return err
 	}
 
+	config, err := configEnvVariables(platform)
+	if err != nil {
+		return err
+	}
+
+	containerName := fmt.Sprintf("%s-skupper-controller", config.username)
+
+	isContainerAlreadyRunningInPodman, err := IsContainerRunning(containerName, types.PlatformPodman)
+	if err != nil {
```

Comment:
This is assuming that both podman and docker are installed on the system, which might not always be true.
Maybe safer to ignore the error if either podman or docker is not available?

Reply:
done